src/main/java/jdbchomework/console/CompaniesMenu.java

```diff
@@ -0,0 +1,87 @@
+package jdbchomework.console;
+
+import jdbchomework.controller.CompanyController;
+import jdbchomework.controller.ProjectController;
+import jdbchomework.entity.Project;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CompaniesMenu extends AbstractMenu {
+
+    private static final String COMPANY_NOT_FOUND = "Company not found";
+    private static final String PLEASE_ENTER_COMPANY_NAME = "Please enter company name: ";
+    private static final String PLEASE_ENTER_COMPANY_ID = "Please enter company ID: ";
+
+    private CompanyController companyController;
+    private ProjectController projectController;
+
+    public CompaniesMenu(MenuContents menuContents, CompanyController companyController, ProjectController
+        projectController) {
+        super(LoggerFactory.getLogger(CompaniesMenu.class), menuContents);
+        this.companyController = companyController;
+        this.projectController = projectController;
+    }
+
+    public void menu() {
+        mark:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor the code to remove this label and the need for it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ALabelsShouldNotBeUsedCheck)